db-scripts/createdb.sql

```diff
@@ -19,8 +19,8 @@ create table if not exists majors(
 );
 
 create table if not exists users(
-   id serial,
-   created_at timestamp not null default CURRENT_TIMESTAMP,
+   id text not null,
```

Comment:
since you changed user id to text, make sure all user_id foreign keys are also of type text